easybuild/easyconfigs/f/FreeFem++/FreeFem_metis.patch

```diff
@@ -0,0 +1,59 @@
+diff -Naur freefem++-3.45-old/examples++-load/metis.cpp freefem++-3.45/examples++-load/metis.cpp
```

Comment:
@stachon Please include one or two lines of comments above this line describing the patch and include yourself as an author:

```
fix <short problem description, mention relevant compiler errors if there are any>
author: <name> (<affiliation>)
```